go/tests/endpoint/endpoint.go

```diff
@@ -54,6 +54,20 @@ type fn func()
 
 var cleanup fn = func() {}
 
+var cwd, err_cwd = os.Getwd()
+var bin, err_bin = os.Executable()
+
+func init() {
+	if err_cwd == nil && err_bin == nil {
+		bindir := filepath.Dir((bin))
+		if bindir != cwd {
+			os.Chdir(bindir)
+			cwd = bindir
+			Say("Executing out of '%s'", cwd)
```

Comment:
change single quotes around %s formatter to double quotes for consistency (see writeIPC's error ret)